lib/active_record_shards/association_collection_connection_selection.rb

```diff
@@ -1,12 +1,22 @@
 # frozen_string_literal: true
 module ActiveRecordShards
   module AssociationCollectionConnectionSelection
-    def on_slave_if(condition)
-      condition ? on_slave : self
+    def on_replica_if(condition)
+      condition ? on_replica : self
     end
 
-    def on_slave_unless(condition)
-      on_slave_if(!condition)
+    def on_slave_if(*args)
```

Comment:
Can we just use `alias` for the, well, aliases? So we define `def on_replica_if(condition)` and `alias :on_slave_if, :on_replica_if`. I think that will make it easier to clean up later.

Reply:
I think that the goal here is to add a deprecation warning where the `# TODO` is.